### PR TITLE
CXP-1074: Grammarly proposals

### PR DESCRIPTION
**Pull request by tseho**

### CODE REVIEW

File: content/apps/using-openid.md [hunk 1]
```diff
@@ -119,13 +118,13 @@ The ID Token sent by Akeneo PIM contains a signature, and you must verify it to
...
 To validate the signature, you must retrieve the public key available at the
-url `{PIM}/connect/apps/v1/openid/public-key`.
+URL `{PIM}/connect/apps/v1/openid/public-key`.
 
 Then, follow the instructions of the library you are using.
 
 ::: warning
-The pair of private/public keys is regenerated regularly for security reasons. You should always retrieve
-the lastest public key when validating a signature.
+The pair of private/public keys are regenerated regularly for security reasons. It would be best if you always retrieved
```
Comment (LevFlavien) [inline content/apps/using-openid.md:126]: Subject is "The pair" so I'd keep "is"

Comment (LevFlavien) [inline content/apps/using-openid.md:126]: Did you drop "should" to make the instruction more lenient?
To keep the tone formal, we could use "It is recommended to retrieve...". But I don't mind either way.

**Review by LevFlavien — APPROVED** [review]

File: content/apps/using-openid.md (hunk 1 above)
Comment (tseho) [reply content/apps/using-openid.md:126]: I'm not really fan of the changes proposed in this PR.
It's only changes proposed automatically by the grammarly tool.

File: content/apps/using-openid.md [hunk 2]
```diff
@@ -41,12 +40,12 @@ You can consult the list of [available authentication scopes](/apps/access-scope
 
 ## Authenticate after the first Authorization request
 
-If an user is trying to access your App from his Akeneo PIM, and you want to authenticate him, start an
+If a user is trying to access your App from his Akeneo PIM and wants to authenticate him, start an
```
Comment (LevFlavien) [inline content/apps/using-openid.md:43]:
```suggestion
If a user is trying to access your App from his Akeneo PIM and wants to authenticate, start an
```
Though the meaning isn't exactly the same as the original version (before grammarly)

Comment (tseho) [reply content/apps/using-openid.md:43]: Exactly.
For context, when I started fixing some typos in this page, Romain tried to use Grammarly to automatically fix other typos but It changed the meaning of some sentences.

I then separated the changes into 2 PRs:
- easy typos: https://github.com/akeneo/pim-api-docs/pull/547/files (merged)
- controversial changes suggested by Grammarly (current PR)

Comment (tseho) [reply content/apps/using-openid.md:43]: I will close this PR, since I don't know what to do with it.

**Pull request closed without merging**